data/saas/config/ada_chatbot_config.yml

```diff
@@ -0,0 +1,51 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Ada Chatbot
+  type: ada_chatbot
+  description: A sample schema representing the Ada Chatbot connector for Fides
+  user_guide: https://docs.ethyca.com/user-guides/integrations/saas-integrations/ada_chatbot
+  version: 0.1.0
+
+  connector_params:
+    - name: compliance_access_token
+      sensitive: True
+      description: The Data Compliance API value. This value can be seen by going to the Settings --> Integrations within the Admin GUI (https://developers.ada.cx/reference/authentication)
+      label: Data Compliance API
```

Comment:
Is this the correct label for the `compliance_access_token`? The description mentions the "The Data Compliance API" but that doesn't seem right to me

Reply:
I used that name to match what this is called in the Ada chatbot admin interface. I thought it should match what it is called in the spot they hopefully are looking for it?

This is what it looks like 
<img width="1182" alt="Screenshot 2023-11-08 at 7 49 02 AM" src="https://github.com/ethyca/fides/assets/144141312/940d700d-9b10-4ff9-abf9-4405578e5846">